js/details.js

```diff
@@ -27,6 +27,14 @@
       });
 
       window.addEventListener('AutoTheme:palette', this.onPalette);
+      var panel = this.panel;
+      panel.addEventListener('pop', function onPop() {
```

Comment:
Just wondering why you prefer to add the listener each time we call prepareForDisplay instead of adding it once (like: `Details.panel.addEventListener(...)` at the end of the file); this would also remove the need of calling removeEventListener.

Or do you prefer to have as less event handlers as possible at one given moment ?